### PR TITLE
Change Project Structure: Move Figures, Remove PDF Duplicates, and Lint Code

### PR DESCRIPTION
**Pull request by RektPunk**
## Summary
- Moved the figure directory: 
  - The figure directory was relocated to the root folder, and all related paths in the code were updated accordingly.
- Removed duplicate pdfs: 
  - Duplicated PDF files were removed, and the paths for the remaining ones were adjusted across the project.
- Changed code for linting: 
  - Code was updated to comply with linting standards, ensuring better readability and maintainability.
- Edit web code

### CODE REVIEW

File: Solar-Fullstack-LLM-101/05_1_ChromaDB.ipynb [hunk 1]
```diff
@@ -181,14 +181,25 @@
...
-   "execution_count": 8,
+   "execution_count": 1,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/rektpunk/Library/Caches/pypoetry/virtualenvs/cookbook-pEbIoE7q-py3.12/lib/python3.12/site-packages/langchain_upstage/layout_analysis.py:135: UserWarning: UpstageLayoutAnalysisLoader is deprecated.Please use langchain_upstage.document_parse.UpstageDocumentParseLoader instead.\n",
```
Comment (chosh0615) [inline Solar-Fullstack-LLM-101/05_1_ChromaDB.ipynb:191]: Can you remove these lines with your local directory paths?

Comment (RektPunk) [reply Solar-Fullstack-LLM-101/05_1_ChromaDB.ipynb]: Sure. I removed those lines.

**Comment by RektPunk** [comment]
Hey @chosh0615, just a gentle reminder—can you review this PR again?

**Comment by RektPunk** [comment]
I think this is not a priority at the moment, so will close the PR.

**Pull request closed without merging**